fabric-item-api-v1/src/main/java/net/fabricmc/fabric/mixin/item/client/HeldItemRendererMixin.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019, 2020 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.fabricmc.fabric.mixin.item.client;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import net.minecraft.client.render.item.HeldItemRenderer;
+import net.minecraft.item.BowItem;
+import net.minecraft.item.CrossbowItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+@Mixin(HeldItemRenderer.class)
+public class HeldItemRendererMixin {
+
+	@Redirect(method = "renderFirstPersonItem", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/ItemStack;getItem()Lnet/minecraft/item/Item;"))
+	private Item renderFirstPersonItem(ItemStack itemStack) {
+		return itemStack.getItem() instanceof CrossbowItem ? Items.CROSSBOW : itemStack.getItem();
+	}
+
+	@Redirect(method = "renderItem(FLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider$Immediate;Lnet/minecraft/client/network/ClientPlayerEntity;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/ItemStack;getItem()Lnet/minecraft/item/Item;"))
+	private Item renderItem(ItemStack itemStack) {
+		return itemStack.getItem() instanceof CrossbowItem ? Items.CROSSBOW

```

Comment:
Really should be an if statement rather than two tenary operators for readability:

```suggestion
		if (itemStack.getItem() instanceof CrossbowItem) {
			return Items.CROSSBOW; // Return crossbow for rendering
		}

		if (itemStack.getItem() instanceof BowItem) {
			return Items.BOW; // Return bow for rendering
		}

		return itemStack.getItem();
```